wp-content/themes/core/assets/pcss/spacing/_utilities.pcss

```diff
@@ -0,0 +1,30 @@
+/* -------------------------------------------------------------------------
+ *
+ * Aspect Ratio Helpers
+ *
+ * Useful for fixing the height of an element based on it's width.
+ *
+ * Example:
+ * <figure class="wp-block-image s-aspect-ratio-1-1 s-aspect-ratio-cover">
+ *     <img class="wp-post-image" src="..."></img>
+ * </figure>
+ *
+ * ------------------------------------------------------------------------- */
+
+.s-aspect-ratio-cover {
+	height: 100%;
+	width: 100%;
+	object-fit: cover;
+}
+
+.s-aspect-ratio-1-1 {
+	aspect-ratio: 1 / 1;
+}
+
+.s-aspect-ratio-4-3 {
+	aspect-ratio: 4 / 3;
+}
+
+.s-aspect-ratio-16-9 {
+	aspect-ratio: 16 / 9;
+}
```

Comment:
Should these be mixins pulled into helper classes so that we can use them where we don't have access to classes?

Also, I think if you put these classes on a `figure` element the `img` won't inherit the state, so the image wouldn't actually be cover but the figure would. Maybe this can be handled in the image block styling though.

Reply:
Updated the `.s-aspect-ratio-cover` selector to include `.s-aspect-ratio-cover > img` to also allow the `img` within the image block to cover. Otherwise, the image would take it's own form.